frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/AddDataModal/Panels/PanelsHeader.tsx

```diff
@@ -29,6 +27,14 @@ export const PanelsHeader = ({
             c="brand"
           >{t`Manage databases`}</Box>
         )}
+        {showUploadsLink && (
+          <Box
```

Comment:
maybe a `subtle` variant button rather than a box would be more semantic?

Reply:
I was thinking about the UnstyledButton, but it wouldn't change much. This is an anchor, not a button. It's a polymorphic component.